.github/workflows/cicd.yml

```
@@ -0,0 +1,56 @@
+            -   name: Configure AWS credentials
+                uses: aws-actions/configure-aws-credentials@v4
+                with:
+                    aws-access-id-key: ${{ secrets.AWS_ACCESS_ID_KEY }}
+                    aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+                    aws-region: ap-northeast-2
+
+            -   name: Login to Amazon ECR
+                run: |
+                    aws ecr get-login-password | docker login --username AWS --password-stdin $ECR_REGISTRY
+
+            -   name: Build Docker image
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)